src/cell.h

```diff
@@ -260,9 +260,9 @@ struct Cell {
             if (text.stylebits & STYLE_BOLD) style += L"font-weight: bold;";

```

Comment:
could you make a similar change for uses of `stylebits` ?

Reply:
Yes, I can try.